logspout/start.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+# (C) Copyright 2017 FUJITSU LIMITED
+
+/bin/logspout
```

Comment:
I'd add this [one](https://github.com/krallin/tini#using-tini) here, or somewhere else where it fits. But in general I'd go for explicit approach instead of relying on version of the docker engine.

Reply:
Done